containers/ecr-viewer/src/app/admin/program/ProgramForm.tsx

```diff
@@ -430,6 +430,19 @@ const ConfirmationModal = ({
   onConfirm: () => void;
   modalRef: RefObject<ModalRef>;
 }) => {
+  const prevProgramAreas = Array.from(
+    new Set(categoryConditions.map((c) => c.program_area_name)),
+  );
```

Comment:
```suggestion
  const prevProgramAreas = [
    ...new Set(categoryConditions.map((c) => c.program_area_name)),
  ].sort(stringSort);
```

just for consistency with everywhere else